packages/client-ui/src/components/common/WorkflowStep.tsx

```diff
@@ -4,19 +4,20 @@ import Breadcrumbs from '@material-ui/core/Breadcrumbs'
 import Typography from '@material-ui/core/Typography'
 import Divider from '@material-ui/core/Divider'
 
-import { ModuleViewFactory } from '@dharpa-vre/client-core'
-import { StepDesc } from '@dharpa-vre/client-core/src/common/types/kiaraGenerated'
+import { ModuleViewFactory, WorkflowPageDetails } from '@dharpa-vre/client-core'
 
 import useStyles from './WorkflowStep.styles'
 
 type WorkflowStepProps = {
-  projectSteps: StepDesc[]
+  workflowPages: WorkflowPageDetails[]
   activeStep: number
 }
 
-const WorkflowStep = ({ projectSteps, activeStep }: WorkflowStepProps): JSX.Element => {
+const WorkflowStep = ({ workflowPages, activeStep }: WorkflowStepProps): JSX.Element => {
```

Comment:
If we have to replace the word `step` with `page`, then I'd replace `activeStep` with `activePage` or `currentPage` too, for the sake of consistency.